Project1/Index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+  <title>GD2 Project 1</title>
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+  <style>
+    body {
+      font-family: 'Roboto', sans-serif
+    }
+  </style>
+</head>
+
+
+
+<header>
+  <figure>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+      intelligent.</h2>
+    <img src="images/octopus.jpg" alt="A picture of an octopus">
+    <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+      uncertain how the trait evolved. Fred Tanneau/Agence France-Presse — Getty Images</figcaption>
+  </figure>
+
+
+  <h3>By Carl Zimmer</h3>
+
+  <h4>Nov. 30, 2018</h4>
+</header>
+
+
+<body>
+  <p>To demonstrate how smart an octopus can be, Piero Amodio points to a <span>YouTube video</span>. It shows an
+    octopus
+    pulling two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like
+    nesting bowls — and carts them away.</p>
+
+  <p>“It suggests the octopus is carrying these tools around because it has some understanding they may be useful
+    in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of Cambridge
+    in Britain.</p>
+  <aside clearfix>
```

Comment:
Clearfix is a classname and needs to be referencedlike so:

```suggestion
  <aside class="clearfix">
```

Reply:
See note in the CSS about this, too; you can actually delete the classname here.